reaper/eeg_reaper.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+""" SciTran EEG Reaper """
+
+import os
+import sys
+import glob
+import shutil
+import logging
+import argparse
+import datetime
+
+from . import util
+from . import reaper
+
+log = logging.getLogger('reaper.eeg')
+
+FILETYPE = 'eeg'
+
+DESCRIPTION = u"""
+This reaper will monitor a given directory for EEG files created with
+BrainVision Recorder and upload them together with their header (.vhdr)
+and marker (.vmrk) files to a scitran/core-compatible API, including
+creation of the appropriate project/session/acquisition hierarchy.
+
+The hierarchy is inferred from the EEG filename segments separated by
+underscores and/or the directory structure. The acquisition uid may
+be omitted in which case the creation timestamp will be used instead.
+
+Raw Files
+├── group-id_project-label_subject-code_session-uid_acquisition-uid.eeg
+└── group-id
+    └── project-label
+        ├── subject-code_session-uid_acquisition-uid.eeg
+        └── subject-code
+            └── session-uid
+                └── acquisition-uid.eeg
+"""
+
+
+class EEGReaper(reaper.Reaper):
+
+    """EEGReaper class"""
+
+    has_mapkey_arg = False
+    has_opt_arg = False
+
+    def __init__(self, options):
+        path = options.get('path')
+        if not os.path.isdir(path):
+            log.error('path argument must be a directory')
```

Comment:
Per our "new convention" :wink:, all log statements now start with a capital letter.

Please apply throughout; I'm only noting this here.